extensions/amp-addthis/0.1/amp-addthis.js

```diff
@@ -223,6 +223,19 @@ class AmpAddThis extends AMP.BaseElement {
       // Only the component that registers the page view listens for x-frame
       // events.
       this.setupListeners_({ampDoc, loc, pubId: this.pubId_});
+
+      // Create close button for listing tool
+      if (this.element.getAttribute('data-widget-type') === 'messages') {
+        const closeButton = createElementWithAttributes(
+          this.win.document,
+          'div',
```

Comment:
Why `div` instead of `button`? The latter is more semantic and accessible: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Reply:
Hey Choumx, if I use button, it will come with the default button style which we don't need.

Reply:
You can override the default user agent button styles, right?

Reply:
Sure. will do